harc_game_web/harc_game_web/urls.py

```diff
@@ -26,16 +26,17 @@
     path('logout/', LogoutView.as_view(), name='logout'),
     path('login/', LoginView.as_view(template_name='users/login.html'), name='login'),
 
-    # tasks <str:tab>
+    # tasks
     path('tasks/', TaskView.as_view(), name='tasks', kwargs={'tab': None}),
-    path('tasks/<str:tab>', TaskView.as_view(), name='tasks'),
+    path('tasks/<slug:tab>', TaskView.as_view(), name='tasks'),
     path('completed_tasks/', list_completed_tasks, name='list_completed_tasks'),
-    path('completed_tasks/new/', add_completed_task, name='add_completed_task'),
+    path('completed_tasks/new/', add_completed_task, name='add_completed_task', kwargs={'task_id': None}),
+    path('completed_tasks/new/<slug:task_id>', add_completed_task, name='add_completed_task'),
     path('completed_tasks/edit/<slug:documented_task_id>', edit_completed_task, name='edit_completed_task'),
-    path('completed_tasks/fav/<int:id>', fav_task, name='fav_task'),
-    path('completed_tasks/fav/<int:id>/<str:tab>', fav_task, name='fav_task'),
-    path('completed_tasks/unfav/<int:id>', unfav_task, name='unfav_task'),
-    path('completed_tasks/unfav/<int:id>/<str:tab>', unfav_task, name='unfav_task'),
+    path('completed_tasks/fav/<slug:id>', fav_task, name='fav_task'),
+    path('completed_tasks/fav/<slug:id>/<slug:tab>', fav_task, name='fav_task'),
+    path('completed_tasks/unfav/<slug:id>', unfav_task, name='unfav_task'),
+    path('completed_tasks/unfav/<slug:id>/<slug:tab>', unfav_task, name='unfav_task'),
```

Comment:
dlaczego zmiana z int na slug?

Reply:
A w sumie to teraz nie wiem dlaczego...

Reply:
Tab dlatego, by zeslugować nazwie kategorii. Musze jeszcze dodać to w kodzie. Zapomniałem. A id z rozpędu...